cumulusci.yml

```diff
@@ -499,6 +503,17 @@ flows:
                 options:
                     namespaced: False
 
+    qa_org_2gp:
+        steps:
+            2:
+                flow: config_2gp
+            3:
+                task: None
+            4:
+                flow: config_qa
```

Comment:
@MatthewBlanski do we need config_qa again here since deploy_post and update_admin_profile are part of  config_2gp?

Reply:
Ah nvm, I see coonfig_qa has other stuff too.

Reply:
Can we move, step 3 to 7 in config_qa to config_2gp, so that we don't have to run deploy_post and update_admin_profile twice(once in each flow)?

Reply:
My thoughts on this were to not replicate a second flow for this specific scenario and let us continue on config_qa. The bug with the installer will be fixed soon so having a double non-destructive run for the time being felt like the lowest LOE. If we want to go that route we can, though it will need some additional cleanup.